src/libponyrt/sched/cpu.c

```diff
@@ -227,10 +227,11 @@ uint32_t ponyint_cpu_count()
 }
 
 uint32_t ponyint_cpu_assign(uint32_t count, scheduler_t* scheduler,
-  bool pin, bool pinasio, bool pinpat)
+  bool pin, bool pinasio, bool pinpat, uint32_t* tracing_cpu, bool pin_tracing_thread)
```

Comment:
thoughts on reversing the boolean and the u32 here?

Reply:
other than this small thing, i think this looks good.

Reply:
no thoughts.. want them switched?

Reply:
weird.. i didn't see:

> other than this small thing, i think this looks good.

previously when i added my comment..

will push a change with the order switched as requested soonish..

Reply:
done